src/content/docs/adduserattribute-python-agent-api.mdx

```diff
@@ -2,6 +2,7 @@
 title: add_user_attribute (Python agent API)
 contentType: apiDoc
 template: basicDoc
+shortDescription: '(Deprecated.) Adds a custom attribute to a transaction. '
```

Comment:
Could you run a `.trim()` at for each `shortDescription` to get rid of the unneeded space at the end?